packages/installer/upgrade.sh

```diff
@@ -71,6 +71,18 @@ find_upgrade_channel() {
     if [ -z "$UPGRADE_IMAGE" ]; then
         UPGRADE_IMAGE="system/cos"
     fi
+
+    if [ -n "$UPGRADE_RECOVERY" ] && [ $UPGRADE_RECOVERY == true ] && [ -n "$RECOVERY_IMAGE" ]; then
```

Comment:
This allows to set a `RECOVERY_IMAGE` into `/etc/cos-upgrade-image` allowing to override the default image/package for recovery

Reply:
Added in the docs